src/Material/Tooltip.elm

```diff
@@ -0,0 +1,523 @@
+module Material.Tooltip
+  exposing
+    ( Model
+    , defaultModel
+    , Msg(..)
+    , update
+    , view
+    , render
+    , DOMState
+    , left
+    , right
+    , top
+    , bottom
+    , default
+    , large
+    , Property
+    , onMouseEnter
+    , onMouseLeave
+    , attach
+    , onEnter
+    , onLeave
+    , mdl
+    )
+
+{-| From the [Material Design Lite documentation](https://getmdl.io/components/index.html#tooltips-section):
+
+> standard HTML tooltip as produced by the `title` attribute. A tooltip consists
+> of text and/or an image that clearly communicates additional information about
+> an element when the user hovers over or, in a touch-based UI, touches the
+> element. The MDL tooltip component is pre-styled (colors, fonts, and other
+> settings are contained in material.min.css) to provide a vivid, attractive
+> visual element that displays related but typically non-essential content,
+> e.g., a definition, clarification, or brief instruction.
+>
+> Tooltips are a ubiquitous feature of most user interfaces, regardless of a
+> site's content or function. Their design and use is an important factor in the
+> overall user experience. See the tooltip component's Material Design
+> specifications page for details.
+
+See also the
+[Material Design Specification](https://material.google.com/components/tooltips.html).
+
+Refer to [this site](http://debois.github.io/elm-mdl#/tooltips)
+for a live demo.
+

```

Comment:
Add docs here indicating that users must _both_ use `attach` to install handlers on the target element, as well as call `render` on the following sibling element.